criu/timer.c

```diff
@@ -29,17 +29,8 @@ static inline int decode_itimer(char *n, ItimerEntry *ie, struct itimerval *val)
 		return -1;
 	}
 
-	if (ie->vsec == 0 && ie->vusec == 0) {
```

Comment:
This code handles the case when the timer is armed, but it_value is set to 0:
https://man7.org/linux/man-pages/man2/setitimer.2.html
```
 If  both fields of it_value are zero, then this timer is currently disarmed (inactive).
```